### PR TITLE
Change db.addUser to db.createUser in mongo setup instructions.

### PR DESCRIPTION
**Pull request by mattcollier**
I believe the additional changed lines are due to my editor removing some trailing spaces.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ To setup an admin user on mongodb:
 
     Enter the mongo shell
     use admin
-    db.addUser( { user: "admin", pwd: "password", roles: [ "clusterAdmin", "readWriteAnyDatabase", "userAdminAnyDatabase", "dbAdminAnyDatabase"] } )
+    db.createUser( { user: "admin", pwd: "password", roles: [ "clusterAdmin", "readWriteAnyDatabase", "userAdminAnyDatabase", "dbAdminAnyDatabase"] } )
```
Comment (msporny) [inline README.md]: Which version of MongoDB are you using? Your fix is most likely correct, and if so, we need to update it across all of our projects. I'm on MongoDB 2.4 at home and at work. I don't know if Bedrock works with MongoDB 2.6 yet.

/cc @dlongley

Comment (mattcollier) [inline README.md]: I have the current stable release 3.0.3 which is likely the issue.  I'll install 2.4 for now.

Comment (mattcollier) [inline README.md]: db.addUser is the correct method for v2.4.9.

**Comment by dlongley** [comment]
bedrock-mongodb docs say: "Versions 2.4 and 2.6 are currently supported." Which is true. We'd need to make sure that the driver we're using does some backwards compatibility abstraction with `db.createUser` for 2.4, otherwise making this change will cause it to no longer work with 2.4. I do think we can drop 2.4 support once we have 3.x support working, but we haven't even tried that yet.

**Pull request closed without merging**